### PR TITLE
Create a queued notebook deployment

### PR DESCRIPTION
**Pull request by Ladicle**

### CODE REVIEW

**Comment by Ladicle** [comment]
@kkaneda could you please review this? thanks!

**Comment by kkaneda** [comment]
Ah, sorry!

File: dispatcher/internal/dispatcher/notebook.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+			WithTemplate(corev1apply.PodTemplateSpec().
+				WithLabels(labels).
+				WithSpec(corev1apply.PodSpec().
+					WithContainers(corev1apply.Container().
+						WithName("jupyterlab").
+						WithImage(nb.Image).
+						WithImagePullPolicy(corev1.PullIfNotPresent).
+						// TODO: rethink authentication method
+						WithCommand("start-notebook.py").
+						WithArgs("--IdentityProvider.token=''").
```
Comment (kkaneda) [inline dispatcher/internal/dispatcher/notebook.go:122]: Just a comment. We might later decide to set a token when we use ingress to expose the endpoint.

**Review by kkaneda — APPROVED** [review]
LG!

**Pull request merged**